Tests/DownloadTests.swift

```diff
@@ -309,44 +305,35 @@ class DownloadResponseTestCase: BaseTestCase {
         XCTAssertNil(response?.error)
     }
 
-    func testThatDownloadingFileAndMovingToDestinationThatIsOccupiedThrowsError() {
-        do {
-            // Given
-            let directoryURL = testDirectoryURL.appendingPathComponent("some/random/folder")
-            let directoryCreated = FileManager.createDirectory(at: directoryURL)
-
-            let fileURL = directoryURL.appendingPathComponent("test_output.json")
-            try "random_data".write(to: fileURL, atomically: true, encoding: .utf8)
+    func testThatDownloadingFileAndMovingToDestinationThatIsOccupiedThrowsError() throws {
+        // Given
+        let directoryURL = testDirectoryURL.appendingPathComponent("some/random/folder")
+        let directoryCreated = FileManager.createDirectory(at: directoryURL)
 
-            let expectation = self.expectation(description: "Download should complete but fail to move file")
-            var response: DownloadResponse<URL?, AFError>?
+        let fileURL = directoryURL.appendingPathComponent("test_output.json")
+        try "random_data".write(to: fileURL, atomically: true, encoding: .utf8)
 
-            // When
-            AF.download("https://httpbin.org/get", to: { _, _ in (fileURL, []) })
-                .response { resp in
-                    response = resp
-                    expectation.fulfill()
-                }
+        let expectation = self.expectation(description: "Download should complete but fail to move file")
+        var response: DownloadResponse<URL?, AFError>?
```

Comment:
You could probably collapse these as well if you wanted.

Reply:
Collapse what?

Reply:
Sorry, `DownloadResponse` -> `AFDownloadResponse`.

Reply:
We could do this for all uses of the `Response` types. For now, we can keep them consistently the base type. If we want to use the AF types, we can make the changes all at once I think.